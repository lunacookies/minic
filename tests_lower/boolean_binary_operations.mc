func main {
	var t = 1
	var f = 0
	var eq = t == t
	var ne = t != f
	var lt = f < 39
	var gt = 4 > t
	var le = t*5 <= t
	var ge = t <= f-1
}
