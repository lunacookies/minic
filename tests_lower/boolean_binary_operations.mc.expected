func main
	var t i64;
	var f i64;
	var eq i64;
	var ne i64;
	var lt i64;
	var gt i64;
	var le i64;
	var ge i64;
	{
		set t = 1;
		set f = 0;
		set eq = (t == t);
		set ne = (t != f);
		set lt = (f < 39);
		set gt = (4 > t);
		set le = ((t * 5) <= t);
		set ge = (t <= (f - 1));
	}
tests_lower:37..53: warning: unused variable
tests_lower:55..71: warning: unused variable
tests_lower:73..89: warning: unused variable
tests_lower:91..106: warning: unused variable
tests_lower:108..126: warning: unused variable
tests_lower:128..146: warning: unused variable
